cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-permanent-temporary-location-at-once.cy.js

```diff
@@ -0,0 +1,125 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import Locations from '../../../support/fragments/settings/tenant/location-setup/locations';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
+
+let user;
+const barcode = `barcode-${getRandomPostfix()}`;
+const item = {
+  instanceName: `instanceName-${getRandomPostfix()}`,
+  firstBarcode: barcode,
+};
+const itemBarcodesFileName = `itemBarcodes_${getRandomPostfix()}.csv`;
+const changedRecordsFileName = `*-Changed-Records-${itemBarcodesFileName}`;
+const matchedRecordsFileName = `*Matched-Records-${itemBarcodesFileName}`;
+const previewFileName = `*-Updates-Preview-${itemBarcodesFileName}`;
+const testData = {
+  folioInstances: InventoryInstances.generateFolioInstances(),
+  servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+  requestsId: '',
+};
+
+describe('Bulk Edits', () => {
```

Comment:
please follow this notation, as other tests in the folder
`describe('bulk-edit', () => {
  describe('in-app approach', () => {`